src/main/webapp/app/entities/quiz-exercise/quiz-exercise-detail.component.ts

```diff
@@ -465,6 +470,28 @@ export class QuizExerciseDetailComponent implements OnInit, OnChanges, OnDestroy
         return isGenerallyValid && areAllQuestionsValid;
     }
 
+    /**
+     * @function containExplanation
+     * @desc Shows the explanation warning
+     */
+    containExplanation(): boolean {
+        if (this.pendingChanges() && this.validQuiz()) {
+            return true;
+        }
+
```

Comment:
I still don't understand this function. It does not check anything related to explanations!

Reply:
we want to move that to the markdown editor branch since it depends on the location of this function